client/src/components/inputs/DateInput.svelte

```diff
@@ -0,0 +1,10 @@
+<script>
+  export let value;
+  export let label;
+  export let required = 'required';
+</script>
+
+<div class="form-group">
+  <label for={label}>{label}</label>
```

Comment:
Ca marchera avec des labels en un mot, mais quid des labels "Titre de la page" par exemple ? Même si les id avec espaces sont valides (https://developer.mozilla.org/fr/docs/Web/HTML/Attributs_universels/id) malgré la spec original (https://www.w3.org/TR/2011/WD-html5-20110525/elements.html#the-id-attribute) est-ce qu'il ne faudrait pas les passer en kebab case pour être tranquille ?
Un util ou quelque chose du jour à importer dans les inputs ?

Reply:
Tout à fait :+1: 
Puis j'aime bien les kebab :p

Reply:
Ou sinon utiliser des `shortid` https://www.npmjs.com/package/shortid ?